moveit_ros/planning_interface/move_group_interface/include/moveit/move_group_interface/move_group.h

```diff
@@ -38,6 +38,8 @@
 #ifndef MOVEIT_MOVE_GROUP_INTERFACE_MOVE_GROUP_
 #define MOVEIT_MOVE_GROUP_INTERFACE_MOVE_GROUP_
 
+#include <moveit/macros/class_forward.h>
+#include <moveit/macros/deprecation.h>
```

Comment:
The `#include <moveit/macros/deprecation.h>` came from a different PR. There's no harm I suppose, but the include is probably not needed.